hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -792,6 +793,7 @@ protected void deleteCacheEntry(Data key) {
         }
     }
 
+    @SuppressFBWarnings("WMI_WRONG_MAP_ITERATOR")
```

Comment:
FindBugs complained about `keySet()` used ineffectively. But the usage is quite complex and this should be a false positive. At least I will not optimize this in this PR.